content/blog/kamailio-developpers-meeting-2019-kemi

```diff
@@ -0,0 +1,46 @@
+Title: Kamailio developpers meeting 2019 - kemi framework
+Date: 2019-11-21
+Author: Mathias WOLFF
+Category: kamailio
+Tags: wazo-platform, development, kamailio, KEMI
+Slug: kamailio-developpers-meeting-2019-kemi
+Status: published
+
+
+During 2 days, the [Kamailio](https://www.kamailio.org) developers (The well-known SIP proxy) met in Dusserdolf in the wonderful places of SipgateDE.
+
+
+An important subject was approached in detail: KEMI (Kamailio Embedded Interface). KEMI start with the 5th Kamailio version.
+
+
+KEMI is a framework allowing to use a language other than native language to program our Kamailio engine. With KEMI, we can use Javascript, LUA, Python, LUA, Ruby and Squirrel.
+
+
+The advantages in the use of KEMI are numerous.
+
+First of all, the use of the standard language used by our developers guarantees an enhanced code quality.
```

Comment:
First of all, the use of a standard language used and well known by developers guarantees an enhanced code quality.